src/components/AddTaskButton.jsx

```diff
@@ -20,18 +24,18 @@ const ParentContainer = styled.div`
 const ButtonAddTask = styled(Button)`
   background-color: #b8e3fd;
   border-color: #b8e3fd;
-  color: black;
+  color: #000;
   height: 64px;
   width: 100%;
```

Comment:
Aside from using themes with Styled Components. maybe, import these colors from a constant? you seem to re-use the same color alot. :)